packages/components/datetime/DateTime.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'DateTime'
+type: 'component'
+status: 'stable'
+slug: /components/date-time/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/datetime'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-datetime--basic'
+typescript: ./src/DateTime.tsx,./src/RelativeDateTime.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+In this package, there are two components (`DateTime` and `RelativeDateTime`) that will return a date following [Forma 36 guidelines](https://f36.contentful.com/guidelines/date-and-time/).
+
+## DateTime
+
+The DateTime component will format any date to a human friendly format and wrap it in a `<time>` tag.
+It allows four different forms of display that can be controlled by changing the `format` prop:
+
+| `format` option | Result                 |
+| --------------- | ---------------------- |
+| `full`          | 12 Aug 2020 at 8:00 AM |
+| `day`           | 12 Aug 2020            |
+| `time`          | 8:00 AM                |
+| `weekday`       | Wed, 12 Aug            |
+
+### Examples
+
+```tsx
+// import { DateTime } from '@contentful/f36-datetime';
+
+<DateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<DateTime date={new Date()} />; // passing a JS Date
+<DateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
+```
+
+All the usages above will result in 'Tue, 17 Aug 2021 at 3:45 PM'
+
+## RelativeDateTime
+
+Provides a relative date string for resolutions ranging from seconds up through years. Should be used when absolute date and time is unnecessarily specific.
+
+### Examples
+
+If we consider today as August 18th of 2021 and we use pass the following values to the Component
+
+```tsx
+// import { RelativeDateTime } from '@contentful/f36-datetime';
+
+<RelativeDateTime date="2021-08-17T15:45" />; // passing a Timestamp as an ISOString
+<RelativeDateTime date={new Date()} />; // passing a JS Date
+<RelativeDateTime date={1629240300000} />; // passing a Unix Epoch in milliseconds
+```
+
+The component will display `a day ago`
+
+### Passing a `baseDate`
+
+Normally, the component compare your `date` with today’s date. But maybe you need to render a relative date
+that is compared with some other day that is not today. You can do that by using `baseDate`:
+
+```tsx
+// import { RelativeDateTime } from '@contentful/f36-datetime';
+
+<RelativeDateTime baseDate="2021-08-17T15:45" date="2021-08-18T15:45" />
+```
+
+The component will display `in a day`
+
+### Using `isRelativeToCurrentWeek`
+
+Still using the last example. If you only need a relative date if the given date is within the current week,
+that is possible by using the boolean `isRelativeToCurrentWeek`
```

Comment:
My writing is bad! that's why I need people reviewing my docs for real

So, I'm trying to say that if you want to show "Yesterday at 10:00 PM"
but if the day is more than seven days ago you want to show "12 Aug 2021"

Then you need to pass `isRelativeToCurrentWeek ={true}`

But it's so hard to explain it when I write it!!! do you have suggestions?

Reply:
Yeah, it's hard to explain in a few words, maybe smth like:

> You can set `isRelativeToCurrentWeek`  to `true` if you want to indicate that the date is relative only within a week. And if the difference is more than one week, just show the date without relativeness. It will compare the date to today or `baseDate` if provided.

And it will be useful to add one more example with the date that has difference more than 7 days.

Reply:
Good idea! Thanks